### PR TITLE
LPD-33334 Add Playwright Setup relevant rule

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:sf

File: test.properties [hunk 1]
```diff
@@ -1107,6 +1106,16 @@
 
     test.batch.names[playwright-compile-rule][relevant]=playwright-compile-jdk8
 
+    #
+    # Playwright Setup Rule
+    #
+
+    modified.files.includes[playwright-setup-rule][relevant]=**
+
+    playwright.test.project[playwright-setup-rule][relevant][playwright-js-tomcat90-mysql57-jdk8]=wem-site-setup
```
Comment (ealonso) [inline test.properties:1115]: @sandrodw3 we should change wem with page-management

File: modules/apps/layout/layout-seo-web/src/main/resources/META-INF/resources/js/seo/openGraph.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ import {previewSeoFireChange} from './PreviewSeoEvents';
 
 export default function ({namespace, uploadOpenGraphImageURL}) {
 	const openGraphImageButton = document.getElementById(
-		`${namespace}openGraphImageButton`
+		`${namespace}openGraphImageBustton`
```
Comment (ealonso) [inline modules/apps/layout/layout-seo-web/src/main/resources/META-INF/resources/js/seo/openGraph.js:12]: @sandrodw3 Bustton?

File: test.properties (hunk 1 above)
Comment (sandrodw3) [reply test.properties:1115]: I al ready told this to @veroglez and @victorg1991 but it's too long 😔

**Comment by sandrodw3** [comment]
ci:test:page-management-playwright

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:page-management-playwright

**Comment by sandrodw3** [comment]
ci:stop

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:page-management-playwright

**Comment by sandrodw3** [comment]
ci:forward:force

**Pull request closed without merging**